tests/requirements.txt

```diff
@@ -40,6 +38,8 @@ pytest-xdist
 requests >=2.18.4,<3
 responses
 ruamel.yaml >=0.11.14,<0.18
+setuptools >=31.0.1
```

Comment:
Do we need this?

Reply:
probably not, we need to review our dependencies and remove any that are no longer in use